packages/TelegramClientTests-Misc.package/TCTMMocks.class/class/mockMutedChatEvent.st

```diff
@@ -0,0 +1,20 @@
+chat event
+mockMutedChatEvent
+
+ 	^ (JsonObject newFrom: {
+ 			'@type' -> 'chat'.
+ 			'chat' -> (JsonObject newFrom: {
+ 				'id' -> self mockBasicGroupChat id.
+ 				'title' -> 'MutedMockChat'.
+ 				'type' -> (JsonObject newFrom: {
+ 					'@type' -> 'chatTypeBasicGroup'.
+ 					'basic_group_id' -> self mockBasicGroupChat groupId
+ 				}).
+ 				'permissions' -> (JsonObject newFrom: {
+ 					'@type' -> 'can_send_messages'
+ 				}).
+ 				'notification_settings' -> (JsonObject newFrom: {
+ 					'mute_for' -> 42
+ 				}).
+ 			})
+ 		})
```

Comment:
I think it would be better to parameterize this Event to remove the code duplication in mockUnMutedChatEvent